in-cluster/default-kurl.yaml

```diff
@@ -4,6 +4,14 @@ metadata:
   name: default
 spec:
   collectors:
+    # Note: apiserver logs are owned by root so for this collector
+    # to succeed it requires sudo privileges for the user
+    - copyFromHost:
```

Comment:
won't it not require root because it's running in a pod? (and the pod will be root?)

Reply:
I've removed the comment.